### DISCUSSION TITLE
Flexible Content fields does not give an instance of Main Resource class

### DISCUSSION THREAD

**Discussion by felixmensah321** [laravel/nova-issues#5399] [Q&A] [answered]
I have a Product resource.
Within the resource I have this:
Flexible::make('test', 'test')
->addLayout('', 'wysiwyg', [
Select::make('Type of Guide', 'type')->options($this->getDropDownData('GuideType', 'name'))->rules('required')
->fillUsing(function ($request, $model, $attribute, $requestAttribute) {
dd($model);
})
$model is not giving me the instance of the Product resource but an instance of Whitecube\NovaFlexibleContent\Layouts\Layout

Is there a way to get around this where I get the instance of the Product Resource?

**Comment by davidhemphill** [comment, marked as answer]
Best to post this on the Whitecube Nova Flexible Content issues page. We do not manage that package: https://github.com/whitecube/nova-flexible-content/issues
